packages/ui/src/components/Button/Button.tsx

```diff
@@ -13,6 +13,29 @@ import type { SxProps, Theme } from '@mui/material/styles';
 
 export type ButtonType = 'outlined' | 'primary' | 'secondary';
 
+type ButtonColor =
+  | 'error'
+  | 'info'
+  | 'inherit'
+  | 'primary'
+  | 'secondary'
+  | 'success'
+  | 'warning';
+
+type ButtonVariant = 'contained' | 'outlined' | 'text';
+
+const buttonTypeToColor: Record<ButtonType, ButtonColor> = {
+  outlined: 'secondary', // We're treating this as a secondary
+  primary: 'primary',
+  secondary: 'secondary',
+} as const;
+
+const buttonTypeToVariant: Record<ButtonType, ButtonVariant> = {
+  outlined: 'outlined',
+  primary: 'contained',
+  secondary: 'contained',
+} as const;
```

Comment:
Can we use MUI's types instead of redefining them here?

```suggestion
const buttonTypeToColor: Record<ButtonType, _ButtonProps['color']> = {
  outlined: 'secondary', // We're treating this as a secondary
  primary: 'primary',
  secondary: 'secondary',
} as const;

const buttonTypeToVariant: Record<ButtonType, _ButtonProps['variant']> = {
  outlined: 'outlined',
  primary: 'contained',
  secondary: 'contained',
} as const;
```